common/src/stack/command/stack/commands/set/host/interface/mac/__init__.py

```diff
@@ -47,11 +47,16 @@ def run(self, params, args):
 		if not len(args):
 			raise ArgRequired(self, 'host')
 
-		for host in self.getHostnames(args):
```

Comment:
I'd like to see a comment here about *why* we're moving to only one host at a time.